modin/pandas/test/dataframe/test_window.py

```diff
@@ -206,6 +206,13 @@ def test_fillna_downcast():
     df_equals(modin_df, result)
 
 
+def test_fillna_4660():
```

Comment:
out of curiosity, what is the convention for deciding where this test goes?  i wouldn't have guessed this file

Reply:
Good question, I just found similar tests in this file and added my test here. Do you have any suggestions where to move these tests? I would gladly do it.

Reply:
in pandas this would live in pandas/tests/frame/methods/test_fillna.py; i expect implementing that file-per-method setup in modin would make a lot of noise

Reply:
So, in the file there are no windowing operations according to https://pandas.pydata.org/docs/user_guide/window.html#overview.

I think if we give it a different name, it will already be better. For example, `test_window.py` -> `test_methods.py`. With an increase in the number of tests, it will be possible to divide it into files in the future. What do you think about?

Reply:
`test_methods` feels to generic, can we determine a subset of methods we test here? or maybe split `fillna` in a separate test file...

Reply:
Let's discuss it here https://github.com/modin-project/modin/issues/4688